src/docs/sphinx/dev_guide/tpl_build.rst

```diff
@@ -0,0 +1,372 @@
+
+   $ brew install autoconf automake bzip2 clingo cmake diffutils fmt gcc gettext gnu-sed graphviz hwloc lapack libx11 llvm@19 m4 make ninja open-mpi openblas pkg-config readline zlib
+
+If you plan to install the developer tools, you should also run:
+
+.. code-block:: bash
+
+   $ brew install cppcheck doxygen
+   $ ln -s /opt/homebrew/opt/llvm@19/bin/clang-format /opt/homebrew/bin/clang-format
+
+If you have installed Homebrew using the default installation prefix, most packages will be accessible through the prefix ``/opt/homebrew``.
+Note for Intel-based Macs, the installation prefix is ``/usr/local``. If you set a custom prefix or aren't sure what the prefix is, run ``brew --prefix``.
+For the rest of this section, we will assume the prefix is ``/opt/homebrew``.
+Some packages are not linked into this prefix to prevent conflicts with MacOS-provided versions.
+These will only be accessible via the prefix ``/opt/homebrew/opt/[package-name]``.
+Homebrew will warn about such packages after installing them.
+
+In order for the correct compilers to be used for the installation, you should also add the bin directory for LLVM clang to your path in your ``.bash_profile``, ``.bashrc``, or ``.zshrc``, etc.
+This is also useful for a few additional packages:
+
+.. code-block:: bash
+
+   $ export PATH="/opt/homebrew/opt/llvm@19/bin:/opt/homebrew/opt/m4/bin:/opt/homebrew/opt/gnu-sed/libexec/gnubin:$PATH"
+
+.. note::
+
+    We provide a basic Ubuntu 24 Spack environment file in ``scripts/spack/configs/linux_ubuntu_24`` that
+    may work for most people. If you want to try using that, skip to :ref:`building_tpls-label`
+    below and use this command line option instead ``--spack-env-file=scripts/spack/configs/macos_sonoma_aarch64/spack.yaml``. You will likely
+    need to update the versions of packages to match the versions installed by Homebrew. The versions for all installed packages can be listed via
+    the command ``brew list --versions``.
+
+.. note::
+    The invocation of ``uberenv.py`` is slightly modified from the standard instructions below
+    in order to force the use of the Homebrew-installed MPI and compilers. The spec command line option
+    should be ``--spec="^openmpi@5 %clang@19"`` and to build with devtools and profiling enabled,
+    change the spec to ``"+devtools+profiling ^openmpi@5 %clang@19"``
+
+Optionally, you can install the developer tools via ```pip``. This step is only required if you wish to use Serac's developer tools.
+In order to use Python devtools, you will need to create a Python venv. This is much more reliable than having Spack install 20+ Python packages.
+In this example, we are using the builtin Python in ``/usr/bin``, but it is possible to use a version installed from Homebrew or elsewhere.
+
+Next, you will need to install wheel and Sphinx:
+
+.. code-block:: bash
+
+   python3 -m venv --system-site-packages venv
+   source venv/bin/activate
+   pip install wheel
+   pip install sphinx
```

Comment:
Originally, this section also had

```
   git clone git@github.com:LLNL/ATS.git --branch 7.0.105
   pip install ATS/
```

But I guess a user isn't going to run integration tests, especially now that they're disabled, so it's probably OK to leave out.

Reply:
I figured we would put it back in when we re-enable integration tests.